tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -69,111 +42,37 @@ variables:
   # basic ANSI color support though, so force that in the pipeline
   - name: FORCE_COLOR
     value: 1
-  - name: buildDirectory
-    value: "."
+    readonly: true
   - name: testWorkspace
     value: $(Pipeline.Workspace)/test
+    readonly: true
+  - name: artifactPipeline
+    value: $(resources.pipeline.client.pipelineName)
+    readonly: true
+  - name: artifactBuildId
+    value: $(resources.pipeline.client.runID)
+    readonly: true
+  - name: toolAbsolutePath
+    value: $(Build.SourcesDirectory)/tools/telemetry-generator
+    readonly: true
 
 stages:
-  # Install / Build / Test Stage
-  - stage: build
-    displayName: Run tests
+  # Performance unit tests - runtime
+  - stage: perf_unit_tests_runtime
```

Comment:
They could all be parallelized, yeah. I did a quick check and didn't find how to do it when there's nothing to depend on; I'll check again. Maybe adding an empty first stage just so all the test ones depend on it could be a workaround.

Reply:
Ok, found how we do it in other pipelines: `dependsOn: []`. Adding it here.